index.html

```diff
@@ -1922,30 +1938,29 @@ <h3>Forms</h3>
                             <code style="white-space: nowrap;">LOALPHA *( LOALPHA / DIGIT / "." / "-" )</code>.
                         </span>
                         <span class="rfc2119-assertion" id="arch-op-wellknown-compare">
-                            Well-known operation types MUST be compared character by character in a case-insensitive fashion.
+                            Well-known operation types MUST be compared using a case-insensitive comparison.
                         </span>
                         The well-known operation types for the Web of Things defined by this specification are given in <a href="#table-operation-types">Table 1</a>.
                     </li>
                     <li>
                         <span class="rfc2119-assertion" id="arch-op-extension">
-                            Extension operation types MAY be chosen by applications.
+                            The set of predefined operation types MAY be augmented by <em>Extension operation types</em> chosen by applications.
                         </span>
                         <span class="rfc2119-assertion" id="arch-op-extension-uri">
                             Extension operation types MUST be URIs [[!RFC3986]] that uniquely identify the type.
                         </span>
                         <span class="rfc2119-assertion" id="arch-op-extension-comparison">
-                            Extension operation types MUST be compared as strings (after converting to URIs if serialised in a different format)
-                            character by character in a case-insensitive fashion.
+                            Extension operation types MUST be compared compared as strings using a case-insensitive comparison.
                         </span>
                         <span class="rfc2119-assertion" id="arch-op-extension-lowercase">
-                            Because of this, all-lowercase URIs SHOULD be used for extension operation types.
+                            Nevertheless, all-lowercase URIs SHOULD be used for extension operation types.
                         </span>
                     </li>
                 </ul>
                 <p>
                     <span class="rfc2119-assertion" id="arch-op-request-method">
-                    The request method MUST identify one method of
-                    the standard set of the protocol identified by the
+                    The request method MUST identify one operation type of
+                    the standard set of operation types of the protocol identified by the
```

Comment:
The operation type (op) is different from the method.
Please revert to "method".